<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="frameLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/background/img/jinghai2.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_distance">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>0</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Jinghai Vision</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_calibration">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>720</y>
      <width>270</width>
      <height>25</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="cornerBtn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>find corner</string>
     </property>
    </widget>
    <widget class="QPushButton" name="caliBtn">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>calibration</string>
     </property>
    </widget>
    <widget class="QPushButton" name="numpic">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>0</y>
       <width>30</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="captureBtn">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>0</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>capture</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_camera">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>720</y>
      <width>760</width>
      <height>25</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="zoominBtn">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>0</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>zoom in</string>
     </property>
    </widget>
    <widget class="QPushButton" name="zoomoutBtn">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>0</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>zoom out</string>
     </property>
    </widget>
    <widget class="QPushButton" name="focusfarBtn">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>0</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>focus far</string>
     </property>
    </widget>
    <widget class="QPushButton" name="focsnearBtn">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>0</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>focus near</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stopBtn">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>0</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>stop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startBtn">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>0</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>start</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_focus">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="setfocusBtn">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>0</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>set focus</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_focus">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>0</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="recordBtn">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>0</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>record</string>
     </property>
    </widget>
    <zorder>zoominBtn</zorder>
    <zorder>focusfarBtn</zorder>
    <zorder>focsnearBtn</zorder>
    <zorder>stopBtn</zorder>
    <zorder>startBtn</zorder>
    <zorder>zoomoutBtn</zorder>
    <zorder>lineEdit_focus</zorder>
    <zorder>setfocusBtn</zorder>
    <zorder>label_focus</zorder>
    <zorder>recordBtn</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_serialport">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>720</y>
      <width>521</width>
      <height>25</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="numLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>95</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>Serialport Num:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="numComboBox">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>1</y>
       <width>85</width>
       <height>23</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="burdrateLabel">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>0</y>
       <width>70</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string> Burd Rate:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="burdrateComboBox">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>0</y>
       <width>85</width>
       <height>23</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="databitsLabel">
     <property name="geometry">
      <rect>
       <x>335</x>
       <y>0</y>
       <width>65</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string> Data Bits:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="serialportOpenBtn">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>0</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>open</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_databits">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>0</y>
       <width>40</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="value">
      <number>8</number>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_indicator">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>4</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/indicator/img/red.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_camera_ptz">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>100</y>
      <width>641</width>
      <height>25</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="inBtn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>in</string>
     </property>
    </widget>
    <widget class="QPushButton" name="outuBtn">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>out</string>
     </property>
    </widget>
    <widget class="QPushButton" name="upBtn">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>0</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>up</string>
     </property>
    </widget>
    <widget class="QPushButton" name="downBtn">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>0</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>down</string>
     </property>
    </widget>
    <widget class="QPushButton" name="leftBtn">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>0</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>left</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rightBtn">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>0</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>right</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stopBtn_2">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>0</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>stop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startBtn_2">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>0</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>start</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="serialportBtn">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>26</y>
      <width>108</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>serialport</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cameraBtn">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>52</y>
      <width>108</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>camera</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calibrateBtn">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>78</y>
      <width>108</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>calibration</string>
    </property>
   </widget>
   <widget class="QPushButton" name="laserLightBtn">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>104</y>
      <width>108</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>open light</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rangeFinderBtn">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>130</y>
      <width>108</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>laser rangefinder</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
